core/src/types.ts

```diff
@@ -25,7 +25,8 @@ export type TPaginatedQueryResult<Model extends Json> = {
 };
 
 export type TGraphqlPaginatedQueryResultOptions = {
-  name: string;
+  name?: string;
```

Comment:
One could type this slightly better with a union and intersection type but to be honest I felt this is good enough.